.build/images/Quartz64/quartz64_defconfig

```diff
@@ -336,6 +341,12 @@ CONFIG_IP_SET_HASH_NETPORTNET=m
 CONFIG_IP_SET_LIST_SET=m
 CONFIG_IP_SET_MAX=256
 CONFIG_IP_VS=m
+CONFIG_IP_VS_PROTO_TCP=y
+CONFIG_IP_VS_PROTO_UDP=y
+CONFIG_IP_VS_RR=m
+CONFIG_IP_VS_NFCT=y
+CONFIG_IP_VS_FTP=n
+CONFIG_IP_VS_PE_SIP=n
```

Comment:
Why do you disable those explicitly? They are not defined (not even as "is not set") in our final kernel config, so it would not change something, but RPi and Debian kernels have the modules for those enabled. So, I guess either the lines can be removed, or we add the modules as well.